model-optimizer/extensions/front/split_normalizer.py

```diff
@@ -40,11 +41,17 @@ def run_before(self):
     def find_and_replace_pattern(self, graph: Graph):
         for node in graph.get_op_nodes(squeeze_axis=True):
             name = node.soft_get('name', node.id)
-            assert node.has_valid('axis'), 'Unknown axis to squeeze for node {}'.format(name)
             for out_port in node.out_ports().values():
-                squeeze_node = create_op_with_const_inputs(graph, Squeeze, {1: np.array(node.axis)},
-                                                           {'name': name + '/Squeeze_'})
-                out_port.get_connection().insert_node(squeeze_node)
+                if node.has_valid('axis'):
+                    squeeze_node = create_op_with_const_inputs(graph, Squeeze, {1: np.array(node.axis)},
+                                                               {'name': name + '/Squeeze_'})
+                    out_port.get_connection().insert_node(squeeze_node)
+                elif node.is_in_port_connected(1):
```

Comment:
Nit: Should we make it possible to manipulate this axis input port idx?

Reply:
No, why you ask? Port is 1 since Split has axis at port == 1.

Reply:
If we consider this transformation could work for any operation it would be useful for the future. But never mind lets leave it as it is.